gui/src/components/Taipy/Input.tsx

```diff
@@ -98,7 +98,6 @@ const Input = (props: TaipyInputProps) => {
     return (
         <Tooltip title={hover || ""}>
             <TextField
-                margin="dense"
```

Comment:
You're modifying the default behavior here
margin dense will only be applied when stylekit is used ...

Reply:
Yup, in fact I just took conscience of that and it's already fixed in my last commit